docs/core/tools/csproj.md

```diff
@@ -19,13 +19,26 @@ see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/m
 
 ## Implicit package references
 Metapackages are now implicitly referenced based on the target framework specified in the `<TargetFramework>` or `<TargetFrameworks>` property of your project file. 
-If the target framework is `netcoreap1.x`, the proper version of the `Microsoft.NETCore.App` metapackage is referenced. 
+If the target framework is `netcoreapp1.x`, the proper version of the `Microsoft.NETCore.App` metapackage is referenced. 
```

Comment:
The last change in terms of paragraph structure: Combine the three lines to make one paragraph. Remove "Otherwise" leaving that sentence to start with "If the target ..."

Sorry for all of the text juggling ... it's nice if we get this tidied up a bit.

I suggest making the whole thing look like this ...
```
## Implicit package references
Metapackages are implicitly referenced based on the target framework specified in the `<TargetFramework>` or `<TargetFrameworks>` property of your project file. `<TargetFrameworks>` is ignored if `<TargetFramework>` is specified, independent of order.

If the target framework is `netcoreapp1.x`, the proper version of the `Microsoft.NETCore.App` metapackage is referenced. If the target framework is `netstandard1.x`, the proper version of the `NetStandard.Library` metapackage is referenced. As far as the rest of the behavior is concerned, the tools work as expected and most of the gestures will remain the same (for example, `dotnet restore`).
```